main/modes/mfpaint/paint_browser.c

```diff
@@ -644,3 +656,65 @@ void imageBrowserButton(imageBrowser_t* browser, const buttonEvt_t* evt)
         }
     }
 }
+
+static const char* browserActionToString(imageBrowserAction_t action)
+{
+    switch (action)
+    {
+        case BROWSER_EXIT:
+        {
+            return dialogOptionCancelStr;
+        }
+
+        case BROWSER_OPEN:
+        {
+            return toolWheelLoadStr;
+        }
+
+        case BROWSER_SAVE:
+        {
+            return dialogOptionSaveAsStr;
+        }
+
+        case BROWSER_DELETE:
+        {
+            return deleteStr;
+        }
+
+        default:
+        {
+            return NULL;
+        }
+    }
+}
+
+static uint8_t browserActionToStringLen(imageBrowserAction_t action)
+{
+    switch (action)
+    {
+        case BROWSER_EXIT:
+        {
+            return dialogOptionCancelStrLen;
+        }
+
+        case BROWSER_OPEN:
+        {
+            return toolWheelLoadStrLen;
+        }
+
+        case BROWSER_SAVE:
+        {
+            return dialogOptionSaveAsStrLen;
+        }
+
+        case BROWSER_DELETE:
+        {
+            return ARRAY_SIZE(deleteStr);
+        }
+
+        default:
+        {
+            return 0;
+        }
+    }
+}
```

Comment:
Could this function be replaced with `strlen()`? Also why is `BROWSER_DELETE` different than the others?